itest/rfq_test.go

```diff
@@ -546,13 +546,41 @@ type rfqTestScenario struct {
 	CarolTapd *tapdHarness
 }
 
+// rfqTestScenarioOpts is a struct that holds options related to creating an rfq
+// test scenario.
+type rfqTestScenarioOpts struct {
+	oracleServerAddr string
+}
```

Comment:
it would be better here if we could specify different oracle servers for alice, bob, and carol.